### PR TITLE
katello-host-tools 3.5.0

### PR DESCRIPTION
**Pull request by jlsherrill**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/client/katello-host-tools/katello-host-tools.spec [hunk 1]
```diff
@@ -6,16 +6,16 @@
 
 %{!?python_sitelib: %global python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
 
-%global build_agent (0%{?suse_version} == 0)
+%global build_agent (0%{?suse_version} == 0) && ((0%{?fedora} > 28) || (0%{?rhel} > 0))
```
Comment (jlsherrill) [inline packages/client/katello-host-tools/katello-host-tools.spec]: This update adds support for newer dnf versions which isn't compatible with old, we made the decision to just drop agent support for fedora 27 and 28 as part of this.

**Comment by evgeni** [comment]
@jlsherrill could you please incorporate the changes from https://github.com/theforeman/foreman-packaging/pull/3547 here?

**Comment by jlsherrill** [comment]
@evgeni done!

**Comment by jlsherrill** [comment]
@evgeni hrmm, any idea about the repoclosure issue?

**Comment by evgeni** [comment]
Yeah... We added --newest to the closure call last week, and the fc28 version is newer than el7, so el7 is ignored but required. We just shouldn't run closure for multiple dists in one call. I'd ignore it for now.

**Pull request merged**